From 7742b8614ccb003afb1cf3b894e56cbf3eb40a47 Mon Sep 17 00:00:00 2001
From: thanhbui <thanh.bui.fz@renesas.com>
Date: Tue, 28 Jul 2020 15:04:00 +0700
Subject: [PATCH 427/504] watchdog: renesas-wdt: Register a restart handler for
 R8A77470

This patch registers a restart handler for R8A77470 (RG/G1C) SoC with
Cortex-A7 CPU cores.

Signed-off-by: thanhbui <thanh.bui.fz@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/watchdog/renesas_wdt.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/watchdog/renesas_wdt.c b/drivers/watchdog/renesas_wdt.c
index 9c705d4..feff595 100644
--- a/drivers/watchdog/renesas_wdt.c
+++ b/drivers/watchdog/renesas_wdt.c
@@ -290,7 +290,8 @@ static int rwdt_probe(struct platform_device *pdev)
 	 * register restart handler base on machine here
 	 * same ARM core architecture (e.g ARM cortex A7) can use same handler
 	 */
-	if (of_machine_is_compatible("renesas,r8a7745")) {
+	if (of_machine_is_compatible("renesas,r8a7745") ||
+		of_machine_is_compatible("renesas,r8a77470")) {
 		priv->restart_handler.notifier_call = rwdt_restart_handler_ca7;
 		/* 255: Highest priority restart handler */
 		priv->restart_handler.priority = 255;
-- 
2.7.4

